docs/src/main/sphinx/connector/elasticsearch.md

```diff
@@ -256,42 +307,46 @@ curl --request PUT \
 }'
 ```
 
-Notice for the `array_string_field` that all the following documents are legal
-for Elasticsearch. See the [Elasticsearch array documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/array.html)
-for more details.
+All the following documents are legal for Elasticsearch with
+`array_string_field` mapping:
 
 ```json
 [
     {
         "array_string_field": "trino"
     },
     {
-        "array_string_field": ["trino","is","the","besto"]
+        "array_string_field": ["trino","is","the","best"]
     },
     {
-        "array_string_field": ["trino",["is","the","besto"]]
+        "array_string_field": ["trino",["is","the","best"]]
     },
     {
-        "array_string_field": ["trino",["is",["the","besto"]]]
+        "array_string_field": ["trino",["is",["the","best"]]]
     }
 ]
 ```
 
+See the [Elasticsearch array
+documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/array.html)
+for more details.
+
 Further, Elasticsearch supports types, such as
 [dense_vector](https://www.elastic.co/guide/en/elasticsearch/reference/current/dense-vector.html),
-that are not supported in Trino. New types are constantly emerging which can
-cause parsing exceptions for users that use of these types in Elasticsearch. To
-manage all of these scenarios, you can transform fields to raw JSON by
-annotating it in a Trino-specific structure in the [\_meta](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-meta-field.html)
+that are not supported in Trino. These and other types can cause parsing
+exceptions for users that use of these types in Elasticsearch. To manage all of
+these scenarios, you can transform fields to raw JSON by annotating it in a
+Trino-specific structure in the
+[\_meta](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-meta-field.html)
 section of the index mapping. This indicates to Trino that the field, and all
-nested fields beneath, need to be cast to a `VARCHAR` field that contains
-the raw JSON content. These fields can be defined by using the following command
-to add the field property definition to the `_meta.presto` property of the
-target index mapping.
+nested fields beneath, need to be cast to a `VARCHAR` field that contains the
+raw JSON content. These fields can be defined by using the following command to
+add the field property definition to the `_meta.presto` property of the target
```

Comment:
Is this still meta.presto still in the Elasticsearch connector?

Reply:
Seems like code supports Trino as well.. so ok to change to trino in docs?

Reply:
For now I changed it to Trino since I believe that code supports that and it is the better choice by now.